nyansat/host/shell/antenny_client.py

```diff
@@ -1,17 +1,25 @@
 # Client middle layer
+import logging
+
 from nyansat.host.shell.terminal_printer import TerminalPrinter
-from nyansat.host.shell.nyan_explorer import NyanExplorer, NyanExplorerError
+from nyansat.host.shell.nyan_explorer import NyanExplorer
+from nyansat.host.shell.errors import NotVisibleError, DeviceNotOpenError, NoAntKontrolError, PyboardError
 
 
 def antkontrol_exception(func):
 
     def wrapper(*args, **kwargs):
         try:
             func(*args, **kwargs)
-        except NyanExplorerError as e:
-            # TODO: Fix to use self.printer (decorator must be part of the AntennyClient class somehow)
+        except NoAntKontrolError as e:
+            print("Please run 'antkontrol start' to initialize the antenna.")
+            logging.error(e)
+        except DeviceNotOpenError as e:
+            print("Not connected to device. Use 'open' first.")
+            logging.error(e)
+        except PyboardError as e:
             print("The AntKontrol object is not responding. Restart it with 'antkontrol start'")
```

Comment:
The `PyboardError` is not specific to the `AntKontrol` object. Why I it printing out `The AntKontrol object is not responding`?